### DISCUSSION TITLE
Is it possible to create a dagster asset that depends on other assets created in another file and that belong to an asset_group?

### DISCUSSION THREAD

**Discussion by jordunar** [dagster-io/dagster#19186] [Q&A] [answered]
Hello,

We are facing a problem because we want to create a dagster asset that depends on some other assets that belong to a group and that are defined in a different file **without having to get the assets one by one, just by specifying the group**. We are trying to use the `SourceAsset` function and the `AssetSelection `together (imported from the dagster library), but we are getting some errors.

This is what we have tried:

`deps=SourceAsset(key=AssetKey(AssetSelection.groups("twins_exasol_import")))`

But we get some errors. Is there any way we can achieve what we want?

Thanks beforehand

**Comment by OwenKephart** [comment, marked as answer]
hi @jordunar -- `AssetSelection` unfortunately cannot be used in that way, but if all of the assets in the group are defined in a single asset / subdirectory, then you could probably use `deps=load_assets_from_package_module(place_where.assets_live)`.
